pymc3/tests/test_variational_inference.py

```diff
@@ -3,14 +3,16 @@
 import operator
 import numpy as np
 from theano import theano, tensor as tt
+from theano.configparser import change_flags
 import pymc3 as pm
 from pymc3 import Model, Normal
 from pymc3.variational import (
-    ADVI, FullRankADVI, SVGD,
+    ADVI, FullRankADVI, SVGD, NF,
```

Comment:
What about having a more meaningful name for the class? NF seems a little overzealous. Maybe `NormFlow` or similar?